moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -479,14 +480,7 @@ void PlanningScene::checkCollision(const collision_detection::CollisionRequest&
                                    collision_detection::CollisionResult& res,
                                    const moveit::core::RobotState& robot_state) const
 {
-  // check collision with the world using the padded version
-  getCollisionEnv()->checkRobotCollision(req, res, robot_state, getAllowedCollisionMatrix());
-
-  if (!res.collision || (req.contacts && res.contacts.size() < req.max_contacts))
-  {
-    // do self-collision checking with the unpadded version of the robot
-    getCollisionEnvUnpadded()->checkSelfCollision(req, res, robot_state, getAllowedCollisionMatrix());
-  }
+  checkCollision(req, res, robot_state, getAllowedCollisionMatrix());
```

Comment:
Nice to see this consolidation! I have to say, I don't understand most of the rest of this PR  (why the namespaces changed and so on).